lib/ui/screen/main.dart

```diff
@@ -305,36 +307,31 @@ class _MyHomePageState extends State<MyHomePage> {
                                     ),
                                     onPressed: () {},
                                   ),
-                                  Text('Home',
+                                  const Text('Home',
                                       style: TextStyle(
                                         fontSize: 20,
                                         fontWeight: FontWeight.normal,
                                       )),
                                 ]),
-                            SizedBox(
+                            const SizedBox(
                               height: 20,
                             ),
                             Expanded(
                               child: AnimatedList(
                                 key: alertProvider.listKey,
                                 initialItemCount: alertProvider.items.length,
                                 itemBuilder: (context, position, animation) {
-                                  if (position < alertProvider.items.length) {
-                                    // last item needs padding for bottom sheet
```

Comment:
this padding causing animation started in lower position lol. I don't see any issue of removing the padding though @WenjiuWang